### ISSUE TITLE
Remove Registration for leaderboards.

### ISSUE THREAD

**Issue by Zly-u** [SuperV1234/SSVOpenHexagon#359]
It's completely redundant for Steam Version and should be removed.
Also, there is no way to restore it. Anyway, SteamID already works as a password.

*I forgor my password so I demand this change or else i hack u all to retrieve my password back уъуъуъуъуъу*

If you still have Leaderboard in the compiled non-Steam version then uuhh... Make it as one of the features that require Steam purchase.

**Comment by SuperV1234** [comment]
I am not sure about this, I think that SteamID can be spoofed... I will do some research on that subject. But I do want to avoid people impersonating others.

**Comment by SuperV1234** [comment]
I've done some research and I have to change a bunch of code related to Steamworks authentication to prevent spoofing. It is not a low-effort change, but something that should be done anyway. Once that's done, I will consider removing registration.

**Comment by Zly-u** [comment]
Glad to hear that, I'm sure a lot of people will appreciate this change.

**Comment by Morxemplum** [comment]
Yeah, considering the fact that it's another password that you have to remember. It's a really annoying process. Especially since people are forgetting their passwords for this leaderboard (including me), and there's no way to reset your password if you forget it.
